crates/build/src/pipelines/importer.rs

```diff
@@ -0,0 +1,141 @@
+use anyhow::Context;
+use std::io::prelude::*;
+use std::path::PathBuf;
+
+pub fn import_audio(path: PathBuf, convert: bool) -> anyhow::Result<()> {
+    let current_dir = std::env::current_dir().context("Error getting current directory")?;
+    let asset_folder_path = current_dir.join("assets");
+    let tomlpath = current_dir.join("assets/pipeline.toml");
+    println!("Importing audio...");
+    println!("Read more about audio usage here: https://ambientrun.github.io/Ambient/reference/audio.html");
+    if !std::path::Path::new(&asset_folder_path).exists() {
+        std::fs::create_dir_all(&asset_folder_path)?;
+    }
+    if !std::path::Path::new(&tomlpath).exists() {
+        std::fs::File::create(&tomlpath)?;
+    }
+
+    let mut file = std::fs::File::open(&tomlpath)?;
+    let mut contents = String::new();
+    file.read_to_string(&mut contents)?;
+
+    let mut data: toml::Value = match toml::from_str(&contents) {
+        Ok(v) => v,
+        Err(_) => toml::Value::Table(toml::map::Map::new()), // if we cannot parse the file, start with a fresh table
+    };
+
+    if let toml::Value::Table(table) = &mut data {
```

Comment:
I think I'd prefer to add `use toml::Value` and `use std::path::Path` etc to reduce the noise in the function body 😊